tests/comparisons/Test/testBakeBehaviorTest.php

```diff
@@ -11,6 +11,13 @@ class ExampleBehaviorTest extends TestCase
 {
 
     /**
+     * Test subject
+     *
+     * @var \App\Model\Behavior\ExampleBehavior
+     */
+    public $Example = null;
```

Comment:
@dereuromark You have a point there. Initialising "empty" properties with `null` is a personal habit of mine, I can remove it if it's considered "wrong" or confusing.